content/guides/kubernetes/contour-to-ingress-and-beyond.md

```diff
@@ -0,0 +1,400 @@
+---
+title:  "Getting Started with Contour - To Ingress and Beyond"
+linkTitle: "Getting Started with Contour - To Ingress and Beyond"
+parent: Contour to Ingress and Beyond
+weight: 2
+topics:
+- Kubernetes
+- Tanzu
```

Comment:
Remove "Tanzu" as a topic and that should fix the broken link check. Topic has to be one of the existing topics.